googleapis/api/experimental.pb.go

```diff
@@ -0,0 +1,57 @@
+// Code generated by protoc-gen-go.
+// source: google/api/experimental/experimental.proto
+// DO NOT EDIT!
+
+package api
```

Comment:
I don't think it **blocks** the PR, since the libraries at beta quality are all handwritten anyway. I'll let others confirm.